test/test_orm.py

```diff
@@ -187,25 +179,24 @@ def test_pose(self):
         self.plan()
         pycram.orm.base.ProcessMetaData().description = "pose_test"
         pycram.task.task_tree.root.insert(self.session)
-        result = self.session.query(pycram.orm.base.Pose).all()
+        result = self.session.scalars(select(pycram.orm.base.Pose)).all()
         self.assertTrue(all([r.position is not None and r.orientation is not None for r in result]))
 
     def test_pose_mixin(self):
         self.plan()
         pycram.orm.base.ProcessMetaData().description = "pose_mixin_test"
         pycram.task.task_tree.root.insert(self.session)
-        result = self.session.query(pycram.orm.base.RobotState).all()
+        result = self.session.scalars(select(pycram.orm.base.RobotState)).all()
         self.assertTrue(all([r.pose is not None and r.pose_id == r.pose.id for r in result]))
 
 
-class ORMObjectDesignatorTestCase(BulletWorldTestCase):
+class ORMObjectDesignatorTestCase(ORMTestSchema, BulletWorldTestCase):
```

Comment:
Inheritance and TestCases are a weird thing together. Most likely, it will execute all your test from the super class again in this instance. Im unsure if this is what should be done. Instead write a DatabaseTestCaseMixin that does not inherit from unittest.TestCase but overwrites the setup and teardown methods. Then each database testcase can inherit from that and no test is executed twice.